contracts/account/ArgentAccount.cairo

```diff
@@ -199,6 +203,9 @@ func supportsInterface{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_che
 // EXTERNAL FUNCTIONS
 /////////////////////
 
+// @dev Initialises the account with the signer and an optional guardian
```

Comment:
i'd add some comment to explain how it should be used, as it might not be straightforward.
My suggestion:
```
// It's mandatory to call this methods to secure the account.
// It's recommended to call this method in the same transaction that deploys the account to make sure it's always initialized
```